deps/pom.xml

```diff
@@ -43,15 +43,6 @@
     </properties>
 
     <dependencies>
-        <dependency>
-            <groupId>commons-codec</groupId>
-            <artifactId>commons-codec</artifactId>
-            <!--
-            Note! Do not increase the version of this dependency
-            Android devices are unable to use versions of this library higher than version 1.2
```

Comment:
I'm still running some tests around Android support, but this change doesn't appear to break anything despite this comment I left a while back